### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by GemmaTuron** [ersilia-os/ersilia-client#1]
We might want to update this @DhanshreeA 

` DEPRECATION: Legacy editable install of Ersilia-Client==0.1.0 from file:///home/gturon/github/ersilia-os/ersilia-client (setup.py develop) is deprecated. pip 25.0 will enforce this behaviour change. A possible replacement is to add a pyproject.toml or enable --use-pep517, and use setuptools >= 64. If the resulting installation is not behaving as expected, try using --config-settings editable_mode=compat. Please consult the setuptools documentation for more information. Discussion can be found at https://github.com/pypa/pip/issues/11457`

**Comment by GemmaTuron** [comment]
This can be a good first issue for a contributor familiar with python packaging.

We need to move from a `setup.py` to a `pyproject.toml`

**Comment by P-Mefut** [comment]
Hi @DhanshreeA, I'll love to work on this... could you please assign it to me?

**Comment by AnushkaShreyam** [comment]
Hi @DhanshreeA, can you please assign me this issue?

**Comment by DhanshreeA** [comment]
@AnushkaShreyam please go ahead.

**Comment by AnushkaShreyam** [comment]
@DhanshreeA Requesting for a pull request review. Thank you!

**Comment by DhanshreeA** [comment]
@AnushkaShreyam could you please mention this issue in the PR as "Closes #2"? This would automatically close the issue when the PR is merged.

**Comment by DhanshreeA** [comment]
@AnushkaShreyam could you run the ersilia client with your changes in a new environment and share result here?

**Comment by AnushkaShreyam** [comment]
Hi @DhanshreeA,

The previous dependencies in the pyproject.toml were taking longer to execute, so I've upgraded them to the latest versions for better performance.

Here is the updated output. Let me know if you need any further adjustments.

Thank you!

![Screenshot 2024-10-08 at 9 05 18 PM](https://github.com/user-attachments/assets/13617151-fd43-42bb-8b91-a05842295ec1)

**Comment by DhanshreeA** [comment]
@AnushkaShreyam this looks good, as a next step could you also actually use the ersilia client and share results? You can find usage instructions in the README.

**Comment by AnushkaShreyam** [comment]
@DhanshreeA Yes, I used the Ersilia client for eos43at, and here are the results. Thank you. 

eos43at.py : 
```
from ersilia_client import ErsiliaClient
ec = ErsiliaClient("https://eos43at-zqx9x.ondigitalocean.app/")
smiles_list = ["CCCCOC", "CCCCCN"]
print(ec.run(smiles_list))
```

![Screenshot 2024-10-09 at 6 08 42 PM](https://github.com/user-attachments/assets/d0b6c6d5-e48b-4b70-bf00-f03ac0b4286c)

**Comment by DhanshreeA** [comment]
Fixed in #2 thanks to @AnushkaShreyam!
